project-docs/javascript-linter-spec.md

```diff
@@ -0,0 +1,163 @@
+# JavaScript reference docs structure specification
+
+This document describes the structure we want to require for JavaScript reference documentation.
+
+## Recipes and ingredients overview
+
+It is expected that pages documenting different sorts of thing will need to contain different elements. For example, the page for the JavaScript `BigInt` class will want to contain lists of links to its members, while the page for the JavaScript `BigInt.asUintN()` static method will want to contain a section describing the method syntax.
```

Comment:
I wasn't sure if it is "sorts of thing**s**"?
This then made me read up https://english.stackexchange.com/questions/68599/this-kind-of-things-vs-these-kinds-of-thing and now I wonder if we can instead say "It is expected that pages documenting different things will need to contain different elements." to avoid this altogether.

Reply:
Well... "different sorts of thing" is supposed to mean, say, methods or properties or classes. "different things" could be, say, `Array`, `BigInt`, `Date` - which are the same sort of thing.

I've tried to clarify things in this commit: https://github.com/mdn/stumptown-content/pull/391/commits/1e3a35e224bbf7e71c6318085cc1ec8b72d5dbca - but might have made it worse!

BTW whenever I read threads like that stackexchange thing (1) I'm so sorry for people who have to learn English and (2) I realise I don't understand any of it.

![me-fail-english](https://user-images.githubusercontent.com/432915/78842176-2cbed700-79b4-11ea-8a9c-6890e61e4aeb.jpg)

Reply:
You made it better :) Thanks for talking about "page types". I often think of page types when I think of recipes, so thanks for mentioning this here.

> BTW whenever I read threads like that stackexchange thing (1) I'm so sorry for people who have to learn English and (2) I realise I don't understand any of it.

Thanks for your sympathy! :smile: This makes me feel less stupid with English.